Idea-Archive/Sources/Presentation/WriteDown/WriteDownVC.swift

```diff
@@ -0,0 +1,218 @@
+        
+        jobOpeningCategoryTitleStackView.addArrangedSubview(jobOpeningCategoryTitleLabel)
+        jobOpeningCategoryTitleStackView.addArrangedSubview(jobOpeningCategorySubTitleLabel)
+    }
+    
+    override func setup() {
+        self.navigationItem.title = "새 글 쓰기"
+        self.navigationController?.navigationBar.topItem?.title = ""
+        self.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "다음으로", style: .plain, target: self, action: #selector(nextTap))
+    }
+    
+    @objc func nextTap() {
+        let chooseMajor = ChooseMajorVC()
+        navigationController?.pushViewController(chooseMajor, animated: true)
+    }
+    
+    override func setLayout() {
+        titleStackView.snp.makeConstraints {
+            $0.top.equalTo(self.view.safeAreaLayoutGuide).inset(24)
+            $0.leading.equalToSuperview().inset(24)
+        }
+
+        categoryButtonStackView.snp.makeConstraints {
+            $0.top.equalTo(titleStackView.snp.bottom).inset(-40)
+            $0.leading.equalToSuperview().inset(24)
+            $0.trailing.equalToSuperview().inset(24)
+        }
+        
+        ideaCategoryButton.snp.makeConstraints {
+            $0.height.equalTo(160)
+        }
+        
+        feedbackCategoryButton.snp.makeConstraints {
+            $0.height.equalTo(160)
+        }
+        
+        jobOpeningCategoryButton.snp.makeConstraints {
+            $0.height.equalTo(160)
+        }
+        
+        ideaCategoryTitleStackView.snp.makeConstraints {
+            $0.leading.equalTo(ideaCategoryButton.snp.leading).inset(16)
+            $0.top.equalTo(ideaCategoryButton.snp.top).inset(16)
+        }
+        
+        ideaImageView.snp.makeConstraints {
+            $0.top.equalTo(ideaCategoryButton.snp.top).inset(28)
+            $0.trailing.equalTo(ideaCategoryButton.snp.trailing).inset(28)
+            $0.width.equalTo(83)
+            $0.height.equalTo(104)
+        }
+        
+        feedbackCategoryTitleStackView.snp.makeConstraints {
+            $0.leading.equalTo(feedbackCategoryButton.snp.leading).inset(16)
+            $0.top.equalTo(feedbackCategoryButton.snp.top).inset(16)
+        }
+        
+        feedbackImageView.snp.makeConstraints {
+            $0.top.equalTo(feedbackCategoryButton.snp.top).inset(30)
+            $0.trailing.equalTo(feedbackCategoryButton.snp.trailing).inset(19)
+            $0.width.equalTo(121)
+            $0.height.equalTo(96)
+        }
+        
+        jobOpeningCategoryTitleStackView.snp.makeConstraints {
+            $0.leading.equalTo(jobOpeningCategoryButton.snp.leading).inset(16)
+            $0.top.equalTo(jobOpeningCategoryButton.snp.top).inset(16)
+        }
+        
+        jobOpeningImageView.snp.makeConstraints {
+            $0.trailing.equalTo(jobOpeningCategoryButton.snp.trailing).inset(26)
+            $0.top.equalTo(jobOpeningCategoryButton.snp.top).inset(28)
+            $0.width.equalTo(105)
+            $0.height.equalTo(105)
```

Comment:
```suggestion
            $0.size.equalTo(105)
```
로 Width랑 Height가 같다면 간단하게 쓸 수 있어요!